package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "watson-developer-cloud",
-  "version": "0.10.3",
+  "version": "0.10.4",
```

Comment:
@germanattanasio I would say that it should be your or me faccepting a PR and then ollowing the RELEASE.md steps. I don't think anyone else has permission to do the `npm publish` step.